### DISCUSSION TITLE
more info about "highly correlated"

### DISCUSSION THREAD

**Discussion by anaflorido** [Data-Centric-AI-Community/fg-data-profiling#960] [Q&A]
Hi, im using pandas profiling for data exploration (within google colab notebook with "!pip install https://github.com/pandas-profiling/pandas-profiling/archive/master.zip"), and the thing is, i cant find any details about "highly correlated" on the library documentation,

i would like to know more info about this thresholds that pandas profiling is using when finds a "highly" or other correlations (like > 0.7? less than 0.3? etc)

![imagen](https://user-images.githubusercontent.com/77502030/153716704-41e7bc2d-ed4c-454c-b7ab-d89a140b11cd.png)

and when i get a full profile, and i cant see the values list of corr either: do you know if there's a flag for this?

![imagen](https://user-images.githubusercontent.com/77502030/153716860-1bf64cfd-d257-48d7-a461-e15d0653c3e9.png)

thanks!

**Comment by sbrugman** [comment]
Starting of with a short explanation of how the alerts are generated. The thresholds for this warning are set per correlation, and their defaults can be found [here](https://pandas-profiling.ydata.ai/docs/master/rtd/pages/advanced_usage.html#correlations) (0.9). 

Sensible values for the threshold may differ per dataset. In practice, the dendrogram under 'missing values' is provides an intuitive view for similar variables in terms of missing values. (Hierarchical clustering of the correlation matrices could be an interesting feature!)

As to your second question, could you please clarify what the question is?
